src/context/__tests__/Context.spec.js

```diff
@@ -157,3 +159,21 @@ describe('state', () => {
     });
   });
 });
+
+describe('request context', () => {
+  describe('#requestContext', () => {
+    it('should be empty when no requestContext provided', () => {
+      const { context } = setup({ requestContext: undefined });
+
+      expect(context.requestContext).toBeUndefined();
+    });
+
+    it('should support get requestContext', () => {
+      const requestContext = { req: {}, res: {} };
+
+      const { context } = setup({ requestContext });
+
+      expect(context.requestContext).toBe(requestContext);
```

Comment:
`toBe(requestContext)` or `toBe({ req: {}, res: {} })` ?
Which is more specific?

Reply:
It should be same object and not some other objects which equal to it.